app/grants/views.py

```diff
@@ -568,6 +568,33 @@ def grant_fund(request, grant_id, grant_slug):
         }
         return TemplateResponse(request, 'grants/shared/error.html', params)
 
+    is_team_member = False
+    if grant.admin_profile == profile:
```

Comment:
this would be good to put in a helper function so we can easily find out is_team_member in other views

Reply:
donee